File: leetcode-problems/problem-3232/solution.cpp
// Problem: 3232. Find if Digit Game Can Be Won

class Solution {
public:
    bool canAliceWin(vector<int>& nums) {
        int singleSum, doubleSum;
        for(auto i: nums){
            if(i > 9) doubleSum += i;
            else singleSum += i;
        }
        return singleSum != doubleSum;
    }
};

File: leetcode-problems/problem-3232/solution.py
# Problem: 3232. Find if Digit Game Can Be Won

class Solution:
    def canAliceWin(self, nums: List[int]) -> bool:
        singleSum, doubleSum = 0, 0
        for i in nums:
            if i > 9:
                doubleSum += i
            else:
                singleSum += i

        return singleSum != doubleSum